### DISCUSSION TITLE
File uploads in different upload speeds

### DISCUSSION THREAD

**Discussion by lyriad** [livewire/livewire#3083] [7. Bugs (archived)]
### Description
I've been doing some tests uploading a 10mb video and I found that it takes a long time to upload to the storage on relatively decent up speeds.

I have a 5Mbit (625Kbps) upload speed and it uploads fast, but when I test with 2Mbit (250Kbps) and 3Mbit (375Kbps) upload speeds the video takes very long (5-6 minutes) to upload. Also sometimes when the video finally uploads it fails the validation and just says "failed to upload". I've tried with slower connections and as expected it takes longer and always fails validation.

To limit my network connection I've used NetLimiter and the Developer Tools in Chrome.

As a solution to the upload time I've implemented a progress bar using the upload events described in the documentation [here](https://laravel-livewire.com/docs/file-uploads#js-hooks), but there's also the error of validation. The rules I'm using are required|filled|mimes:mp4|max:100000.

### Steps to reproduce

**Context**
- Livewire version: 1.2.0
- Laravel version: 6.2
- Browser: Chrome 84
- [Link to the video](http://mirrors.standaloneinstaller.com/video-sample/lion-sample.m4v)
- Video size: 10mb
- Video format: MP4

**Comment by nuernbergerA** [comment]
Hey Manuel,

could you confirm the same behavior with other filetypes at the same size?

**Comment by lyriad** [comment]
Hey @nuernbergerA , sorry for the delay.

Did some tests with M4A, FLV, AVI and MKV videos. Also with a 9.55mb MP3 audio file and a 8.67mb PNG image. All took around 6 minutes to upload, give or take 25 seconds on average. 

All tests were using the same 2Mbit (250Kbps) and 3Mbit (375Kbps) upload speeds.

**Comment by austenc** [comment]
@lyriad Which storage driver are you using?

**Comment by squishythejellyfish** [comment]
👋 Oh Hi! I'm Squishy, the friendly jellyfish that manages Livewire issues.

I see this issue has been closed.

Here in the Livewire repo, we have an "issues can be closed guilt-free and without explanation" policy.

If for ANY reason you think this issue hasn't been resolved, PLEASE feel empowered to re-open it.

Re-opening actually helps us track which issues are a priority.

Reply "REOPEN" to this comment and we'll happily re-open it for you!

(More info on this philosophy here: https://twitter.com/calebporzio/status/1321864801295978497)

**Comment by lyriad** [comment]
REOPEN

**Comment by lyriad** [comment]
@austenc i'm using the public driver both locally and on the development server.

**Comment by fflnvb** [comment]
Are there any updates on this issue? I'm badly struggling with uploading large files via Livewire. The upload times and response by the progress indicator do not match at all.
